### PR TITLE
Added Eureka Server Registry

### PR DESCRIPTION
**Pull request by anandmnair**

### CODE REVIEW

File: registry-server/pom.xml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+            <artifactId>jsonassert</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>info.cukes</groupId>
+            <artifactId>cucumber-junit</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>info.cukes</groupId>
+            <artifactId>cucumber-spring</artifactId>
+            <scope>testß</scope>
```
Comment (vincent-fuchs) [inline registry-server/pom.xml]: what is that strange character after "test" ? doesn't it cause any issue ?

**Comment by anandmnair** [comment]
Thanks Vincent. I removed the extra special character from the pom.xml.

File: registry-server/pom.xml (hunk 1 above)
Comment (paul58914080) [inline registry-server/pom.xml]: I wonder how travis CI allowed it.

Comment (paul58914080) [inline registry-server/pom.xml]: @vincent-fuchs I think you might have not turned on travis CI. Can you please turn it on (if not)

**Comment by vincent-fuchs** [comment]
@paul58914080 , travis CI added, thanks !

**Pull request merged**